kernel/data-pipeline/dialect/postgresql/src/test/java/org/apache/shardingsphere/data/pipeline/postgresql/ingest/wal/WALEventConverterTest.java

```diff
@@ -89,6 +94,24 @@ private void initTableData(final DumperConfiguration dumperConfig) {
         }
     }
     
+    @Test
+    public void assertConvertBeginXidEvent() {
+        BeginXidEvent beginXidEvent = new BeginXidEvent(100);
+        beginXidEvent.setLogSequenceNumber(new PostgreSQLLogSequenceNumber(LogSequenceNumber.valueOf("0/0")));
+        Record record = walEventConverter.convert(beginXidEvent);
+        assertTrue(record instanceof PlaceholderRecord);
+        assertThat(((WALPosition) record.getPosition()).getLogSequenceNumber().asLong(), is(0L));
+    }
+    
+    @Test
+    public void assertConvertCommitXidEvent() {
+        CommitXidEvent commitXidEvent = new CommitXidEvent(1, 3468L);
+        commitXidEvent.setLogSequenceNumber(new PostgreSQLLogSequenceNumber(LogSequenceNumber.valueOf("0/0")));
```

Comment:
Could we use more meaningful LSN? Since the default LSN value might be 0 too (in openGauss driver LogSequenceNumber), the unit test might not take effect